adafruit_apds9960/apds9960.py

```diff
@@ -143,7 +150,13 @@ class APDS9960:
     _proximity_persistance = RWBits(4, APDS9960_PERS, 4)
 
     def __init__(
-        self, i2c, *, address=0x39, integration_time=0x01, gain=0x01, rotation=0
+        self,
+        i2c: I2C,
+        *,
+        address: int = 0x39,
+        integration_time: int = 0x01,
+        gain: int = 0x01,
+        rotation: int = 0
     ):
 
         self.buf129 = None
```

Comment:
This buffer being declared as `None` but then getting used later in `gesture()` is a bit weird but understandable. Definitely causes a headache for some more aggressive linter configs though.

Not allocating it saves a decent chunk of memory but maybe there's a better way to handle it, like a `gesture_enabled` flag? Outside the scope here but maybe worth raising an issue for if you guys think its worth improving.